<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LP_MainWindow</class>
 <widget class="QMainWindow" name="LP_MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="focusPolicy">
   <enum>Qt::TabFocus</enum>
  </property>
  <property name="windowTitle">
   <string>Smart Fasion</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(30, 30, 30, 255), stop:1 rgba(84, 84, 84, 255));
color: rgb(206, 206, 206);
QToolTip {
	color:red;
    border: 2px solid black;
    padding: 5px;
    border-radius: 3px;
    opacity: 200;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_4">
    <property name="leftMargin">
     <number>5</number>
    </property>
    <property name="topMargin">
     <number>5</number>
    </property>
    <property name="rightMargin">
     <number>5</number>
    </property>
    <property name="bottomMargin">
     <number>5</number>
    </property>
    <item row="0" column="1">
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QFrame" name="frame">
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="LP_OpenGLWidget" name="openGLWidget">
          <property name="focusPolicy">
           <enum>Qt::ClickFocus</enum>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QFrame" name="frame_2">
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="LP_OpenGLWidget" name="openGLWidget_2">
          <property name="focusPolicy">
           <enum>Qt::ClickFocus</enum>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>39</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QMenuBar::item {
    spacing: 8px; /* spacing between menu bar items */
    padding: 5px 10px;
    background: transparent;
    border-radius: 5px;
	margin: 5px;
}
QMenuBar::item:selected {
	color: &quot;white&quot;;
    background: rgba(160, 160, 160, 220);
	padding: 5px;
    border-radius: 5px;
}
QMenu {
    margin: 5px; /* some spacing around the menu */
}
QMenu::item {
    padding: 2px 25px 2px 20px;
    border: 1px solid transparent; /* reserve space for selection border */
}
QMenu::item:selected {
    background: rgba(120, 120, 120, 255);
}
QMenu::item:disabled {
    color:gray;
}</string>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menuImport">
     <property name="title">
      <string>Import</string>
     </property>
     <addaction name="separator"/>
     <addaction name="lp_import_openmesh"/>
    </widget>
    <addaction name="lp_new"/>
    <addaction name="lp_file_open"/>
    <addaction name="lp_file_save"/>
    <addaction name="separator"/>
    <addaction name="menuImport"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuTools"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QDockWidget" name="dockWidget">
   <property name="styleSheet">
    <string notr="true">QDockWidget{
	background: rgb(198, 198, 198);
}</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="QTabWidget" name="tabWidget">
       <property name="styleSheet">
        <string notr="true">QTabWidget::pane { /* The tab widget frame */
    border-top: 2px solid #C2C7CB;
}

QTabWidget::tab-bar {
    left: 5px; /* move to the right by 5px */
}

/* Style the tab using the tab sub-control. Note that
    it reads QTabBar _not_ QTabWidget */
QTabBar::tab {
	color:#333;
    background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                stop: 0 #E1E1E1, stop: 0.4 #DDDDDD,
                                stop: 0.5 #D8D8D8, stop: 1.0 #D3D3D3);
    border: 2px solid #C4C4C3;
    border-bottom-color: #C2C7CB; /* same as the pane color */
    border-top-left-radius: 4px;
    border-top-right-radius: 4px;
    min-width: 8ex;
    padding: 2px 8px;
}

QTabBar::tab:selected, QTabBar::tab:hover {
    background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                stop: 0 #fafafa, stop: 0.4 #f4f4f4,
                                stop: 0.5 #e7e7e7, stop: 1.0 #fafafa);
}

QTabBar::tab:selected {
    border-color: #9B9B9B;
    border-bottom-color: #C2C7CB; /* same as pane color */
}

QTabBar::tab:!selected {
    margin-top: 2px; /* make non-selected tabs look smaller */
}
QToolTip {
	color:orange;
    border: 2px solid black;
    padding: 5px;
    border-radius: 3px;
    opacity: 200;
}
</string>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="tab">
        <attribute name="title">
         <string>Objects</string>
        </attribute>
        <attribute name="toolTip">
         <string>Objects in the scene</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout">
         <property name="leftMargin">
          <number>2</number>
         </property>
         <property name="topMargin">
          <number>2</number>
         </property>
         <property name="rightMargin">
          <number>2</number>
         </property>
         <property name="bottomMargin">
          <number>2</number>
         </property>
         <item row="0" column="1">
          <widget class="QSplitter" name="splitter_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <widget class="QTreeView" name="treeView">
            <property name="styleSheet">
             <string notr="true">QHeaderView::section {
    background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1,
                                      stop:0 #818181, stop: 0.5 #707070,
                                      stop: 0.6 #636363, stop:1 #858585);
    color: white;
    padding-left: 4px;
    border: 1px solid #6c6c6c;
}
QTreeView {
	background: transaparent;
	color: black;
	margin:2px;
	background-color: rgba(198, 198, 198,255);
}
QToolTip {
	color:orange;
    border: 2px solid black;
    padding: 5px;
    border-radius: 3px;
    opacity: 200;
}</string>
            </property>
            <property name="editTriggers">
             <set>QAbstractItemView::NoEditTriggers</set>
            </property>
            <property name="alternatingRowColors">
             <bool>false</bool>
            </property>
            <property name="selectionMode">
             <enum>QAbstractItemView::ExtendedSelection</enum>
            </property>
            <property name="selectionBehavior">
             <enum>QAbstractItemView::SelectItems</enum>
            </property>
           </widget>
           <widget class="QUndoView" name="undoView">
            <property name="toolTip">
             <string>History of commands</string>
            </property>
           </widget>
           <widget class="QProgressBar" name="progressBar">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Maximum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>24</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QProgressBar {
	color:white;
	text-align: center;
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(40, 40, 40, 255), stop:1 rgba(87, 87, 87, 255));
    border: 2px solid grey;
    border-radius: 5px;
}</string>
            </property>
            <property name="value">
             <number>24</number>
            </property>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tab_dockui">
        <attribute name="title">
         <string>Context UI</string>
        </attribute>
        <attribute name="toolTip">
         <string>UI of current function</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <property name="leftMargin">
          <number>2</number>
         </property>
         <property name="topMargin">
          <number>2</number>
         </property>
         <property name="rightMargin">
          <number>2</number>
         </property>
         <property name="bottomMargin">
          <number>2</number>
         </property>
         <item>
          <widget class="QScrollArea" name="scrollArea">
           <property name="widgetResizable">
            <bool>true</bool>
           </property>
           <widget class="QWidget" name="scroll_dockui">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>258</width>
              <height>584</height>
             </rect>
            </property>
            <layout class="QGridLayout" name="gridLayout_3"/>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="lp_new">
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="lp_file_open">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="lp_import_openmesh">
   <property name="text">
    <string>Open Mesh</string>
   </property>
  </action>
  <action name="lp_file_save">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>LP_OpenGLWidget</class>
   <extends>QOpenGLWidget</extends>
   <header>ui/lp_openglwidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
